tests/conftest.py

```diff
@@ -4,10 +4,13 @@
 
 def pytest_addoption(parser):
     parser.addoption("--package", action="store")
+    parser.addoption("--names-to-skip", action="store", default="")
 
 def pytest_generate_tests(metafunc):
     package_name = metafunc.config.getoption("--package")
+    names_to_skip = metafunc.config.getoption("--names-to-skip")
+    names_to_skip = names_to_skip.split(",") if names_to_skip else []
```

Comment:
```suggestion
    names_to_skip = names_to_skip.split(",") if names_to_skip is not None and names_to_skip != "" else []
```

Minor suggestion for explicit conditions here (assuming `.getoption` returns `None` if flag is not passed)

Reply:
default is `""` actually, so probably best to just check that one.